api/types.go

```diff
@@ -119,6 +119,9 @@ type ShowResponse struct {
 	Parameters string `json:"parameters,omitempty"`
 	Template   string `json:"template,omitempty"`
 	System     string `json:"system,omitempty"`
+	// these fields are used to determine information about the model runtime
+	BaseModel string `json:"base_model,omitempty"`
+	ModelType string `json:"model_type,omitempty"`
```

Comment:
Let's see if we can return some of the manifest fields like `format` here which will tell us if a file is `gguf` or not. If not from the manifest, we can go direct to the model.